### PR TITLE
Allow users to be able to specify directory where raw sequences live.

### PR DESCRIPTION
**Pull request by fnothaft**
We will want this since #139 moves the location of the FASTQ files in our bucket.

### CODE REVIEW

File: src/toil_scripts/adam_gatk_pipeline/align_and_call.py [hunk 1]
```diff
@@ -148,6 +148,11 @@ def build_parser():
                         help = "Skip preprocessing and start running from variant calling. Implies --skip_alignment.",
                         action = 'store_true')
 
+    # what directory are our sequence files in?
+    parser.add_argument('-SD', '--sequence_dir',
```
Comment (hannes-ucsc) [inline src/toil_scripts/adam_gatk_pipeline/align_and_call.py]: Unix convention is that `-SD` is the contraction of `-S -D`. I would either use --sd or ditch it completely.

Comment (fnothaft) [inline src/toil_scripts/adam_gatk_pipeline/align_and_call.py]: Yeah, I think we're at the point where the short arguments aren't very useful. However, we've got a lot of them in here. Would it be OK with you to hold off on changing this for now, and to then go back after we're done and refactor them all? If so, I'll open a ticket to track.

Comment (hannes-ucsc) [inline src/toil_scripts/adam_gatk_pipeline/align_and_call.py]: Absolutely.

**Comment by hannes-ucsc** [comment]
+1

But see https://github.com/BD2KGenomics/toil-scripts/issues/150

**Pull request merged**